packages/bls-kms-local/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@sphereon/ssi-sdk-bls-key-manager",
+  "description": "Sphereon KMS implementation that provides BLS crypto",
+  "version": "0.5.0",
```

Comment:
Fixed

Reply:
Actually, since this is a mono repo using lerna, it should be in line with the base version, meaning 0.5.0. Lerna handles the versions. If this version deviates from the rest of the packages we get in trouble.